### PR TITLE
Add OMIS UK Regional Settings

### PR DESCRIPTION
**Pull request by marcofucci**
This adds a OneToOne model UKRegionalSettings for storing UK manager emails used by OMIS notifications.

I'm not a big fan of the name `UKRegionalSettings` and `ukregion.omis_settings` but I couldn't find a better one.

This also adds the `multi_email_field` dependency defining a field with multiple email addresses separated by new lines. I'm not a big fan of this extra dependency either but anything else seems messier.

### CODE REVIEW

**Comment by reupen** [comment]
Did you consider ArrayField from django.contrib.postgres.fields? I think that ends up being comma-separated in admin, so not quite the same, but that's not much different from writing an email anyway.

Not much of a fan of that package as it doesn't claim compatibility with anything newer than Python 3.5 and Django 1.10.

File: datahub/omis/region/models.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+from multi_email_field.fields import MultiEmailField
+
+from datahub.metadata.models import UKRegion
+
+
+class UKRegionalSettings(models.Model):
+    """OMIS settings for UK Regions."""
+
+    ukregion = models.OneToOneField(
```
Comment (reupen) [inline datahub/omis/region/models.py]: Why `ukregion` rather than `uk_region`?

File: datahub/omis/region/models.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        UKRegion,
+        primary_key=True,
+        on_delete=models.CASCADE,
+        related_name='omis_settings'
+    )
+    manager_emails = MultiEmailField(blank=True)
+
+    class Meta:
+        verbose_name = 'OMIS UK Regional Settings'
```
Comment (reupen) [inline datahub/omis/region/models.py]: Verbose name should be in lower case (except for initialisms, proper nouns etc.) So I would expect `OMIS UK regional settings`.

Comment (marcofucci) [reply datahub/omis/region/models.py]: 👍

File: datahub/omis/region/models.py (hunk 1 above)
Comment (marcofucci) [reply datahub/omis/region/models.py]: 👍

**Comment by marcofucci** [comment]
> Did you consider ArrayField from django.contrib.postgres.fields?
Wow, I didn't notice that it had a base_field, it looks just perfect.

**Comment by marcofucci** [comment]
@reupen much better now, thanks!

**Comment by elcct** [comment]
Indeed much nicer with `ArrayField`.

**Pull request merged**